apollo-ast/src/main/kotlin/com/apollographql/apollo3/ast/internal/ExecutableValidationScope.kt

```diff
@@ -320,11 +315,9 @@ internal class ExecutableValidationScope(
         ))
       }
     }
-  }
 
-  private fun GQLOperationDefinition.validateDeferDirectives() {
-    deferDirectivePathAndLabels.clear()
-    selectionSet.validateDeferDirectives("")
+    if (issues.isNotEmpty()) return
```

Comment:
I'd like to avoid stopping the validation early. This code could be used to display warnings in an editor or something else and we want to have as many issues as possible in a single pass. For an example:

```graphql
{
  computer {
     ...fragment1 @defer
     ...fragment2 @defer
     ...fragment3 @defer(label: 42)
  }
}
```
Ideally, we want an issue to be output about fragment1 and fragment2 (and fragment3 too)

Reply:
Oh ok, that was my way of avoiding to handle `labelStringValue = (label as GQLStringValue).value` later :) I can skip instead.